notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
I don't like the shape of this output. Let's make it look like bow & arrow! <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=46>ReviewNB</a>_